packages/client-api-schema/src/data-types.ts

```diff
@@ -105,14 +105,14 @@ export interface Funds {
 }
 
 export type ChannelStatus =
-  | 'proposed'
-  | 'opening'
-  | 'funding'
-  | 'running'
-  | 'challenging'
-  | 'responding'
-  | 'closing'
-  | 'closed';
+  | 'proposed' // The wallet is storing this channel, but you have not.
```

Comment:
I am a bit confused by "but you have not"

Reply:
Ah, should be "you have not joined the channel" ie. You have not signed any state